pkg/cc/config/config.go

```diff
@@ -100,3 +123,41 @@ func newConfig(reader configReader, overrides ...Overrider) (*Config, error) {
 func jsonDecoderConfig(dc *mapstructure.DecoderConfig) {
 	dc.TagName = "json"
 }
+
+func GetConfigFile() (string, error) {
+	env := os.Getenv("ASERTO_ENV")
+	if env != "" {
+		return env, nil
+	}
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return "", errors.Wrap(err, "failed to determine user home directory")
+	}
+	return filepath.Join(home, ".config", x.AppName, "config.json"), nil
+}
+
+func GetConfigFromFile(filePath string) (*Config, error) {
+	if !FileExists(filePath) {
+		return nil, aErr.NeedLoginErr
+	}
+	dat, err := os.ReadFile(filePath)
+	if err != nil {
+		return nil, err
+	}
+
+	cfg := &Config{}
+	err = json.Unmarshal(dat, cfg)
```

Comment:
Loading the config as json doesn't take environment variables into account. Can't we use `NewConfig()` to load from a file?